themes/bootstrap3/js/search.js

```diff
@@ -329,6 +329,9 @@ VuFind.register('search', function search() {
     fetch(VuFind.path + '/AJAX/JSON?' + queryParams.toString())
       .then((response) => response.json())
       .then((result) => {
+        if (result.error) {
```

Comment:
Is this code related to the rest of the PR, or is this a general bug fix that we might want to merge separately?

Reply:
I don't think there were previously situations where the handler would have returned an error (unless called with bad parameters, but that won't happen in search.js), so this is quite particular to the case here.